scripts/globals/spells/enfeebling_song.lua

```diff
@@ -95,7 +95,10 @@ xi.spells.enfeebling.calculateSongPower = function(caster, spellEffect, basePowe
         if caster:hasStatusEffect(xi.effect.SOUL_VOICE) then
             power = power * 2
         elseif caster:hasStatusEffect(xi.effect.MARCATO) then
-            power = power * 1.5
+            local marcatoEffect = caster:getStatusEffect(xi.effect.MARCATO)
+            if marcatoEffect ~= nil then
+                power = power * marcatoEffect:getPower() / 100
```

Comment:
Could get some overflows here, but if we divide before multiplying that's ok. This change also reflects power of 50 in Marcato
```suggestion
                power = power * (1 + (marcatoEffect:getPower() / 100))
```